### ISSUE TITLE
Small path error

### ISSUE THREAD

**Issue by tarvos21** [nervosnetwork/docs-new#116]
In the  **Guide - Run a CKB Dev Blockchain** seciton 4 **Change the PoW Function to Eaglesong**, there is a line:

> Replace the `miner.workers` section in the `specs/dev.toml` file with the following:

The `specs/dev.toml` here probably should be `ckb-miner.toml`

**Comment by doitian** [comment]
Good catch. Indeed, you have to change both `ckb-miner.toml` and `specs/dev.toml`.

**Comment by doitian** [comment]
Could you please send a PR to fix this?

**Comment by tarvos21** [comment]
> Could you please send a PR to fix this?

The PR is here: #118 

But I am not sure about it. There is no `miner.workers` section in the efault `specs/dev.toml` file generated by the `ckb init` command.

Any suggesstion for the PR? Thanks!
